verification/curator-service/ui/cypress/fixtures/bad_bulk_data.csv

```diff
@@ -0,0 +1,2 @@
+sourceId,sourceEntryId,sex,ageRangeStart,ageRangeEnd,country,locationQuery,latitude,longitude,geoResolution,locationName,dateConfirmedStart,dateConfirmedEnd,url
```

Comment:
Could you delete the examples file now that we have real fixtures in this folder? 
https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/verification/curator-service/ui/cypress/fixtures/example.json

Reply:
Will do!